docs/qcfractal/source/changelog.rst

```diff
@@ -15,25 +15,47 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
-0.12.0 / 2019-MM-DD
+0.12.0 / 2019-11-06
 -------------------
 
+Highlights:
+
+- The ability to handle very large datasets (1M+ entries) quickly and efficiently.
+- Store and compute Wavefunction information.
+- Build and serve views for Datasets.
+- A new GUI dashboard to observe the current state of the server, see statistics, and fix issues.
+
 New Features
 ++++++++++++
 - (:pr:`433`) ``Dataset`` and ``ReactionDataset`` (``interface.collections``) now have a ``download``` method which
```

Comment:
Add pr 462 to this list, remove from Enhancements, don't mention the progress bar.